recipes/catx/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "catx" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/catx-{{ version }}.tar.gz
+  sha256: 31c8b46ae6c8ffe39fe95ef40106e594780a95190ad83c13d28f22b295a4ae44
+
+build:
+  skip: True  # [win or py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - chex
+    - dm-haiku
+    - jax
+    - matplotlib-base
+    - numpy
+    - optax
+    - python
+    - tensorflow =2
+    - typing_extensions  # [py<310]
+
+test:
+  imports:
+    - catx
+  commands:
+    - pytest tests
```

```suggestion
    - pytest tests
    - pip check
    - project_name --help
```
This tests the dependency and the CLI.